Directory.Packages.props

```diff
@@ -10,6 +10,8 @@
 		<PackageVersion Include="MSTest" Version="3.2.2" />
 		<PackageVersion Include="Nito.AsyncEx.Coordination" Version="5.1.2" />
 		<PackageVersion Include="NLog.Web.AspNetCore" Version="5.3.8" />
+		<PackageVersion Include="OpenTelemetry.Exporter.Prometheus.AspNetCore" Version="1.8.0-beta.1" />
```

Comment:
Is there no stable version available for this package?

Reply:
Sadly not. They skipped whole 1.7.0 release with it 😭